corehq/toggles/__init__.py

```diff
@@ -1982,6 +1982,17 @@ def _commtrackify(domain_name, toggle_is_enabled):
     """
 )
 
+RATE_LIMIT_REPEATER_ATTEMPTS = DynamicallyPredictablyRandomToggle(
+    'rate_limit_repeater_attempts',
+    'Apply rate limiting to attempts for data forwarding (repeaters)',
+    TAG_INTERNAL,
+    [NAMESPACE_DOMAIN],
+    description="""
+    In addition to the rate limits based on time spent waiting, these rate limits ensure a project
+    is limited to how many workers they can use at a time.
```

Comment:
When I read the description, it looks like we are limiting the number of workers  a particular domain can use. I think which is not the case.
Can we rephrase it to something like rate limiting the volume of requests that a domain can make?

Reply:
Yeah, conceptually we are limiting the number of workers, but I understand why that isn't helpful. Updated in 744f7a7b8bd7ec5d4fd3d9378476879df075912b